Assets/Scripts/Buddy/BuddyStats.cs

```diff
@@ -204,14 +206,13 @@ public void AdjustHappiness( float deltaHappiness )
 		{
 			_happiness = 0;
 		}
-
-		RecalculateStat();
-
-		if ( _happiness < 0 )
+		else if ( _happiness > 1 )
 		{
-			_happiness = 0;
+			_happiness = 1;
```

Comment:
This seems like something that would be compiled out to me. This guy agrees http://stackoverflow.com/a/16170339